configs/selfsup/eva/metafile.yaml

```
@@ -0,0 +1,46 @@
+Collections:
+  - Name: EVA
+    Metadata:
+      Training Data: ImageNet-1k
+      Training Techniques:
+        - AdamW
+      Training Resources: 16x A100-80G GPUs
+      Architecture:
+        - ViT
+    Paper:
+        URL: https://arxiv.org/abs/2211.07636
+        Title: "EVA: Exploring the Limits of Masked Visual Representation Learning at Scale"
+    README: configs/selfsup/eva/README.md
+
+Models:
+  - Name: eva_mae-sytle_vit-base-p16_16xb256-coslr-400e_1k
```

```suggestion
  - Name: eva-mae-style_vit-base-p16_16xb256-coslr-400e_in1k
```
please also check your all other config names

---

sorry I forget the metafile. Ohter config names have be modified.